### PR TITLE
Update code for getting and saving components form Corgi

### PR DESCRIPTION
**Pull request by hanchuntao**
[OLCS-248](https://issues.redhat.com/browse/OLCS-248)

Fixes OLCS-248

### CODE REVIEW

**Comment by hanchuntao** [comment]
@rhyw @qduanmu @huiwangredhat PTAL

File: openlcs/libs/tests.py [hunk 1]
```diff
@@ -272,57 +272,103 @@ def setUp(self):
...
                     'name': '@jest/fake-timers',
                     'version': '27.2.0',
                     'release': '',
                     'arch': '',
-                    'license': ''
+                    'license': '',
+                    'synced': True
+                }
+            ],
+            'CONTAINER_IMAGE': [
```
Comment (huiwangredhat) [inline openlcs/libs/tests.py:358]: It's not unified with the structure of components in sc_handler. In sc_handler, the components of 'CONTAINER_IMAGE'  should be 'CONTAINER_IMAGE': {}, not [{}]. If like to use this structure, suggest updating get_container_component() in sc_handler.

Comment (hanchuntao) [reply openlcs/libs/tests.py:358]: Updated, use a uniform format

**Comment by huiwangredhat** [comment]
LGTM

**Pull request merged**